lib/galaxy/web/framework/middleware/error.py

```diff
@@ -170,10 +173,11 @@ def __call__(self, environ, start_response):
                 for expect in environ.get("paste.expected_exceptions", []):
                     if isinstance(exc_info[1], expect):
                         raise
+                log.exception("Uncaught Exception")
```

Comment:
This might look redundant, but the `exception_handler` just writes to stderr and bypasses the logging system, so sentry for instance is blind to anything happening in the exception handler.